### PR TITLE
center "404 not found" message horizontally and vertically

### PR DESCRIPTION
**Pull request by MehmetFiratKomurcu**
hi, this is my first open source pr. I am opening this pr for this issue: https://github.com/SimplQ/simplQ-frontend/issues/195 . If there is anything that needs to be change, feel free to comment. This is the screenshot: 
image

### CODE REVIEW

**Comment by maaverik** [comment]
Hey @MehmetFiratKomurcu Kudos on your first PR. It looks like there are things that can still be improved. Let me add some pictures to clarify.

Screenshot from 2020-08-27 00-16-49

My browser's default background color is gray while the site's background color is white, so from the pictures above, you can notice that the div containing the PageNotFound component is not filling up the entire page. This is where we were before, hence the problem with vertical centering.

Screenshot from 2020-08-27 00-15-06

Now this is how the page looks after your change. The message is at the center due to the CSS changes, but now the body is shrunk even further covering only a small part at the top. You can check yourself by inspecting the page using dev tools  and hovering over the body html element.

The problem might be due to some conflicting CSS code in the component. Could you try going over the mixins that are used from https://github.com/SimplQ/simplQ-frontend/blob/new-frontend/simplq/src/styles/utils.scss or messing around with the CSS in dev tools and see if you can figure it out?

**Comment by MehmetFiratKomurcu** [comment]
Hi @maaverik, yes I have a white background that's why I didn't see that. Problem is i used position:absolute. Now I fixed it. Could you review it again please?

File: simplq/src/styles/pageNotFound.module.scss [hunk 1]
```diff
@@ -5,4 +5,5 @@
   @include center-vertically();
   @include text-color();
   font-size: x-large;
+  min-height: 100vh;
 }
```
Comment (maaverik) [inline simplq/src/styles/pageNotFound.module.scss:9]: We've used rem for sizing everywhere else. Could you switch to rem here too?

**Comment by maaverik** [comment]
The 404 page looks good. Great job. Could you look at the create queue page too? I think the CSS change messed that up.
Screenshot from 2020-08-27 01-27-17

Could you also describe what you think the problem was? Just curious.

**Comment by MehmetFiratKomurcu** [comment]
Hi maaverik, asking because didn't understand. In new-frondend branch, body height looks like this, so that gray background should be in there in your browser already, 
image

**Comment by maaverik** [comment]
Oh, you're right. It was there before too. Strangely, the dimensions of the create page are different though, might be because of the border-box, not sure.

Anyway, could you please check if you could get the create page to fill the entire space as well and also just double check if all the pages look fine on mobile view?

**Comment by MehmetFiratKomurcu** [comment]
Hi maaverik, these are the ss of mobile view.
image
image
image
image
and I filled the space for create and join pages, can you review it again please?

**Review by maaverik — APPROVED** [review]

**Comment by maaverik** [comment]
It looks good. Great job. Go ahead and merge

**Comment by MehmetFiratKomurcu** [comment]
I can't find the merge button :D :D Does it have to with this thing? 
image

**Comment by maaverik** [comment]
Yes, click the squash and merge button you see below that :).

**Comment by MehmetFiratKomurcu** [comment]
I think i can't merge because one of these to thing:
image

**Comment by maaverik** [comment]
Oh, I see. I forgot you're not part of the organization. My bad. I'll merge for you.

**Comment by MehmetFiratKomurcu** [comment]
Thank you so much maaverik 🍾

**Comment by maaverik** [comment]
Happy to help. Feel free to take on more issues if you find something you like. We'll add some more well-defined, smaller issues soon.

**Pull request merged**